UI/Web/src/app/nav-header/nav-header.component.ts

```diff
@@ -72,7 +72,7 @@ export class NavHeaderComponent implements OnInit, OnDestroy {
 
   onChangeSearch(val: string) {
       this.isLoading = true;
-      this.searchTerm = val;
+      this.searchTerm = val.trimStart();
```

Comment:
Always trim. Please test to make sure it works in more than just the initial bug. The way I write bugs is very loose. It's expected to do full testing for other potential missed cases.